### PR TITLE
Prevent blacklist circumvention by editing messages

### PR DESCRIPTION
**Pull request by ATechnoHazard**

### CODE REVIEW

**Comment by shanutheflash** [comment]
So, Its like when a person edits his message and uses the blacklisted word the message gets deleted? 🤔

**Comment by Shabier** [comment]
@shanutheflash correct

**Comment by shanutheflash** [comment]
Well, a good one then!

File: tg_bot/modules/blacklist.py [hunk 1]
```diff
@@ -167,7 +167,7 @@ def __stats__():
 ADD_BLACKLIST_HANDLER = CommandHandler("addblacklist", add_blacklist, filters=Filters.group)
 UNBLACKLIST_HANDLER = CommandHandler(["unblacklist", "rmblacklist"], unblacklist, filters=Filters.group)
 BLACKLIST_DEL_HANDLER = MessageHandler(
-    (Filters.text | Filters.command | Filters.sticker | Filters.photo) & Filters.group, del_blacklist)
+    (Filters.text | Filters.command | Filters.sticker | Filters.photo) & Filters.group, del_blacklist, allow_edited = True)
```
Comment (PaulSonOfLars) [inline tg_bot/modules/blacklist.py]: PEP8 - `=` signs in function arguments shouldn't be surrounded by spaces

**Comment by ATechnoHazard** [comment]
Fixed and force pushed :P

**Comment by julianodorneles** [comment]
Shouldn't be `edited_updates`, instead?

**Comment by PaulSonOfLars** [comment]
You're right, `allow_edited` is deprecated

**Comment by PaulSonOfLars** [comment]
You're right, `allow_edited` is deprecated

**Pull request merged**